src/java.desktop/share/classes/javax/swing/JPopupMenu.java

```diff
@@ -758,6 +759,16 @@ public void pack() {
         }
     }
 
+    private Window getMenuInvoker() {
+        if (invoker instanceof Window menuInvoker) {
```

Comment:
[Pattern matching for instanceof](https://openjdk.org/jeps/394) is used.
Maybe we shouldn't use this if we plan to backport this change to the JDK < 16

Reply:
I wouldn't worry about that.